understanding/22/focus-appearance-minimum.html

```diff
@@ -133,6 +133,11 @@ <h4>Unusual shapes and gradients</h4>
                     <figcaption>The focused button with the non-contrasting areas removed, showing that it is a thick indicator that meets the requirements.</figcaption>
                 </figure>
 
+                <p>If an inline-link is broken over two lines the indicator can be split between them. In this case the link element is still treated as one instance.</p>
+                <figure id="focus-indicator-inline-link">
+                    <img src="img/focus-inline-link.png" alt="A paragraph of text with a text link that wraps around. The red focus outline has a gap on the right of one part of the link, and on the left of the other part of the link." />
```

Comment:
For consistency with the preceding paragraph:

```suggestion
                    <img src="img/focus-inline-link.png" alt="A paragraph of text containing a link broken over two lines. The red focus outline has a gap on the right of one part of the link, and on the left of the other part of the link." />
```